docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Get started with Azure for Java using Eclipse
+description: Get started with basic use of the Azure libraries for Java with your own Azure subscription.
+keywords: Azure, Java, SDK, API, authenticate, get-started
+author: roygara
+ms.author: v-rogara
+manager: timlt
+ms.date: 10/30/2017
+ms.topic: get-started-article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: java
+ms.service: multiple
+---
+
+# Get started with the Azure using Eclipse
+
+This guide walks you through setting up a development environment with an Azure service principal. It also provides you with sample code that creates and uses resources in your Azure subscription using the Azure libraries for Java.
```

Comment:
This guide walks you through setting up a development environment and using the Azure libraries for Java. You'll create a service principal to authenticate with Azure and run some sample code that creates and uses Azure resources in your subscription.